lib/remote-ftp.js

```diff
@@ -319,6 +319,31 @@ module.exports = {
 					});
 				});
 			},
+			
+			'remote-ftp:download-tab': function ( e ) {
+				if (!hasProject()) return;
+
+				atom.project.remoteftp.connect();
+				
+				var local = $( event.target ).find( '[data-path]:eq(0)' ).attr( 'data-path' );
```

Comment:
The command `Upload Tab` and `Download Tab` can be called from the command palette (don't think this is intended since we have `Download/Upload Selected` command) which results in the following error:

```
Uncaught TypeError: Arguments to path.join must be strings

At path.js:449

TypeError: Arguments to path.join must be strings
  at Object.posix.join (path.js:449:13)
  at Client.module.exports.Client.toRemote (/Users/jx/Projects/Forks/remote-ftp/lib/client.js:291:15)
  at ConnectorFTP.module.exports.ConnectorFTP.put (/Users/jx/Projects/Forks/remote-ftp/lib/connectors/ftp.js:306:25)
  at Client.<anonymous> (/Users/jx/Projects/Forks/remote-ftp/lib/client.js:398:20)
  at Client.module.exports.Client._next (/Users/jx/Projects/Forks/remote-ftp/lib/client.js:313:21)
  at /Users/jx/Projects/Forks/remote-ftp/lib/client.js:365:11
  at /Users/jx/Projects/Forks/remote-ftp/lib/connectors/ftp.js:151:17
  ...
```